contracts/IdentityOracle.sol

```diff
@@ -83,66 +80,27 @@ contract IdentityOracle is DAOUpgradeableContract, ChainlinkClient, Ownable {
         _onlyOracle();
         lastStartUpdProcInvoked = block.timestamp;
         Chainlink.Request memory req = buildChainlinkRequest(
-            stringToBytes32(JOBID_GET_STATE_HASH),
+            stringToBytes32(JOBID_GET_STATE_HASH_IPFSCID),
```

Comment:
what is this for?
since converting ipfscid to 32bytes string makes it useless, there's no point in using stringtobytes32, simply hash the string using keccak to make it 32bytes

Reply:
fixed by https://github.com/GoodDollar/ChainlinkIdentityOracle/pull/5/commits/f05ce99726a869f1dfaf944456647138eaf25fa3